core/application/account_create.go

```diff
@@ -112,7 +112,7 @@ func (s *Service) setAccountAndProfileDetails(ctx context.Context, req *pb.RpcAc
 		hash, err := bs.UploadFile(context.Background(), spaceID, pb.RpcFileUploadRequest{
 			LocalPath: req.GetAvatarLocalPath(),
 			Type:      model.BlockContentFile_Image,
-		})
+		}, model.ObjectOrigin_user)
```

Comment:
Maybe create DTO for request and add origin there?

It should replicate fields of RpcFileUploadRequest plus contain Origin

Reply:
Done